pages/common/doppler-projects.md

```diff
@@ -0,0 +1,24 @@
+# doppler projects
+
+> Manage Doppler Projects.
+> More information: <https://docs.doppler.com/docs/cli>.
+
+- Get all projects:
+
+`doppler projects`
+
+- Get info for a project:
+
+`doppler projects get {{name/project_id}}`
```

Comment:
Does this require both name and project_id, or only one of them?

```suggestion
`doppler projects get {{name|project_id}}`
```

Reply:
It requires only one of them. Your suggestion `{{name|project_id}}` is the correct one i believe.
Thanks 🫡